test/specs/Hw5/HotelsPOM.js

```diff
@@ -0,0 +1,138 @@
+const { expect, assert } = require("chai");
+
+const moment = require("moment");
+
+const commands = require("./commands");
+
+class homework_5_hotels {
+  
+    now = new moment();
+  
+    lCommands = new commands();
+  
+    //hotels
+  dateForm = "#date_form_field-btn";
+  goingTo = "//button[@aria-label='Going to']";
+  emptyGoingTo = "#destination_form_field";
+  doneButton = '//button[@data-stid = ("apply-date-picker")]';
+  searchButtonHotels = '//button[@id="submit_button"]';
+  locationTitle = '//div[@data-test-id = ("content-location-title")]';
+  dateTitle = '//span[@data-test-id = ("playback-date-range")]';
+
+  async dateFormClick() {
+    let click = await this.lCommands.findWebElement(this.dateForm);
+    return click.click();
```

Comment:
Not sure why you using `return` here in most of the functions but actually you're returning nothing unless it is a getText().
Again for clicking you supposed to use clickWebElement(locator) function of Commands page. In that case it'll be a single line function instead of L23 & L24. Same observation for all click functions.